src/index.ts

```diff
@@ -31,7 +32,12 @@ class DXScannerCommand extends Command {
     const scanPath = args.path || process.cwd();
     cli.action.start(`Scanning URI: ${scanPath}`);
 
-    const container = createRootContainer({ uri: scanPath });
+    const params: ArgumentsProvider = { uri: scanPath };
+    if (authorization) {
```

Comment:
The condition is not necessary, you can add the authorization directly to params on the line above.